### PR TITLE
Quick cult fixes

### PR DESCRIPTION
**Pull request by lilacrhododendron**
:cl:
fix: You can actually purify shades with a Bible now, as intended.
code: Made deconversion just update_body instead of cutting and rebuilding all mob overlays.
spellcheck: Removed an extra exclamation point from the messages for purifying bastard swords and soul stones.
/:cl:

Actually tested the Bible, it works. PRB: No update. Requesting a fat speedmerge since we currently have a feature that does nothing. While I was at it I also fixed the grammar.

### CODE REVIEW

File: code/game/objects/items/storage/book.dm [hunk 1]
```diff
@@ -187,22 +187,22 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
...
-			user.visible_message("<span class='notice'>[user] has purified the [sword]!!</span>")
+			user.visible_message("<span class='notice'>[user] has purified the [sword]!</span>")
 			qdel(sword)
 
 	else if(istype(A, /obj/item/soulstone) && !iscultist(user))
 		var/obj/item/soulstone/SS = A
 		to_chat(user, "<span class='notice'>You begin to exorcise [SS].</span>")
 		playsound(src,'sound/hallucinations/veryfar_noise.ogg',40,1)
-		if(do_after(user, 40, user = SS))
+		if(do_after(user, 40, target = SS))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/book.dm]: me every pr

Comment (subject217) [reply code/game/objects/items/storage/book.dm]: You caught one in the old PR but not this one, so it's your fault >:^)

**Comment by vuonojenmustaturska** [comment]
Your PR is merged, but we do not grant you the label of PRB: No Update

**Comment by subject217** [comment]
Easiest GBP of my life.

**Pull request merged**